bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala

```diff
@@ -115,7 +115,7 @@ class OpenDialog private(db: IDBDatabaseService, auth: KeyChain, vcs: VcsRegistr
   var selection: List[ISPProgram] = Nil
   val filter = new DBProgramChooserFilter(DBProgramChooserFilter.Mode.localAndRemote)
 
-  val MaxM = 50
+  val MaxM = 100
```

Comment:
This seems just for the UI? where is the size enforced?

Reply:
If you look at L367 of `OpenDialog`, you can see that the limit is enforced and programs are only opened if there is enough room in `MaxK`, which is calculated from `MaxM`.

Reply:
Ooops. I changed the version. Grrrr. This perpetual 2016A versioning is annoying.

Reply:
Ok, good to have UI and side effects mixed 😄

Reply:
How so? The UI shows and enforces a limit on how many programs you can have locally. It’s related to keeping the used heap memory down to a reasonable size but it isn’t the same thing as the command line max heap parameter.

Reply:
Btw, the “size” isn’t even heap memory per se but the size of the sum of the corresponding serialized program files.